src/common/product_models/columnar_attribs_common_models.py

```diff
@@ -66,6 +79,18 @@ class AttributesColumnSpec(BaseModel):
         description="The filter strategy for the column if any. Not all column types need "
             + "a filter strategy."
     )] = None
+    display_name: Annotated[str | None, Field(
+        example="Completeness",
+        description="The display name of the column."
+    )] = None
+    category: Annotated[ColumnCategory | None, Field(
```

Comment:
I would make this a string so that non-techie people can add new categories as they wish.

Reply:
👍

Reply:
:+1: